packages/base/rules/best-practices.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  rules: {
+    'no-debugger': 0,
+    'no-alert': 0,
+    'no-await-in-loop': 0,
+    'no-return-assign': ['error', 'except-parens'],
+    'no-restricted-syntax': [
+      2,
+      'ForInStatement',
+      'LabeledStatement',
+      'WithStatement',
+    ],
+    'no-unused-vars': [
+      'error',
+      {
+        vars: 'all',
+        args: 'after-used',
+        ignoreRestSiblings: false,
+      },
+    ],
+    'prefer-const': [
+      'error',
+      {
+        destructuring: 'all',
+      },
+    ],
+    'arrow-body-style': [2, 'as-needed'],
+    'no-unused-expressions': [
+      2,
+      {
+        allowTaggedTemplates: true,
+      },
+    ],
+    'no-param-reassign': [
+      2,
+      {
+        props: false,
+      },
+    ],
```

Comment:
For consiceness can we try and collapse rule definitions to be a single line? Just for this file I think it makes sense, feel free to add `//prettier-ignore`
```suggestion
    'no-param-reassign': [ 2, { props: false } ],
```